util/missingcrypto.txt

```diff
@@ -1124,6 +1124,9 @@ X509V3_section_free(3)
 X509V3_set_conf_lhash(3)
 X509V3_set_nconf(3)
 X509V3_string_free(3)
+X509_ACERT_add_attr_nconf(3)
+X509_ACERT_print(3)
+X509_ACERT_print_ex(3)
```

Comment:
This is unfortunately not allowed. We do not allow any additions to missingcrypto.txt. You will need to add at least some documentation.